crates/core/tedge_mapper/src/c8y/tests.rs

```diff
@@ -49,6 +49,22 @@ async fn mapper_publishes_a_supported_operation_and_a_pending_operations_onto_c8
     // Start SM Mapper
     let sm_mapper = start_sm_mapper(broker.port).await;
 
+    let mut msg = messages
+        .recv()
+        .with_timeout(TEST_TIMEOUT_MS)
+        .await
+        .expect_or("No message received before timeout");
+    dbg!(&msg);
+
+    while !msg.contains("114") {
+        msg = messages
+            .recv()
+            .with_timeout(TEST_TIMEOUT_MS)
+            .await
+            .expect_or("No message received before timeout");
+        dbg!(&msg);
+    }
+
     // Expect both 114 and 500 messages has been received on `c8y/s/us`, if no msg received for the timeout the test fails.
```

Comment:
```suggestion
    // Expect both 118 and 500 messages has been received on `c8y/s/us`, if no msg received for the timeout the test fails.
```
Based on the logic below